src/main/java/com/bp3x/raidbot/commands/RemindCommand.java

```diff
@@ -0,0 +1,65 @@
+package com.bp3x.raidbot.commands;
+
+import com.bp3x.raidbot.commands.util.Event;
+import com.bp3x.raidbot.commands.util.LFGEmbedBuilder;
+import com.bp3x.raidbot.util.RaidBotEmbedBuilder;
+import com.bp3x.raidbot.util.RaidBotRuntimeException;
+import com.jagrosh.jdautilities.command.Command;
+import com.jagrosh.jdautilities.command.CommandEvent;
+import net.dv8tion.jda.api.entities.Member;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Reminder Command to ping accepted players.
+ */
+public class RemindCommand extends Command {
+    private final Logger log = LoggerFactory.getLogger(RemindCommand.class);
+
+
+
+    public RemindCommand() {
+        this.name = "remind";
+        this.help = "Use to remind accepted players";
+        this.arguments = "<eventID> <playerCategory>";
+        this.guildOnly = false;
+        this.ownerCommand = false;
+        this.category = new Category("General");
+
+    }
+
+    @Override
+    protected void execute(CommandEvent event) {
+        log.info("Reminder command by: " + event.getAuthor().getName() + "#" + event.getAuthor().getDiscriminator());
+        log.info("Reminding for " + event.getArgs());
+
+        String[] args = event.getArgs().split("\\s+");
+        if (args.length > 0) {
+            try {
+                // Retrieve event ID data and then add the names of the users in the accepted list
+                Event remindEvent = Event.getEventById(args[0]);
+                if (remindEvent != null) {
+                    if (Event.eventExists(remindEvent.getShortName())) {
+                        StringBuilder ReminderStringBuilder = new StringBuilder();
+                        RaidBotEmbedBuilder pingEmbed = new RaidBotEmbedBuilder();
+                        for (Member player : remindEvent.getAcceptedPlayers()) {
+                            ReminderStringBuilder.append("<@!");
+                            ReminderStringBuilder.append(player.getUser().getId());
+                            ReminderStringBuilder.append(">\n");
```

Comment:
Have we verified that the formatting holds up when there's multiple people being pinged?

Reply:
We haven't. I haven't gotten a chance to hard-code a test for this since the concept of doing that was confusing/ I didn't have time but I can figure it out!